draft-ietf-acme-acme.md

```diff
@@ -1185,12 +1188,16 @@ key that signed the request.
 
 It is up to the server's local policy to decide which names are acceptable in a
 certificate, given the authorizations that the server associates with the
-client's account key.  A server MAY consider a client authorized for a wildcard
-domain if it is authorized for the underlying domain name (without the "*"
-label).  Servers SHOULD NOT extend authorization across identifier types.  For
-example, if a client is authorized for "example.com", then the server should not
-allow the client to issue a certificate with an iPAddress subjectAltName, even
-if it contains an IP address to which example.com resolves.
+client's account key. Servers SHOULD NOT extend authorization across identifier
+types.  For example, if a client is authorized for "example.com", then the
+server should not allow the client to issue a certificate with an iPAddress
+subjectAltName, even if it contains an IP address to which example.com
+resolves.
+
+Wildcard names SHOULD NOT be accepted unless a client is authorized for that
+exact wildcard name. For example, an authorization for `*.*.example.com`
```

Comment:
Only wildcards in the form `"*."` plus a valid FQDN should be supported.  TLS clients generally do not support multiple asterisk characters in a name nor do they support partial label wildcards (e.g. f*.example.com, *g.example.com, or f*g.example.com).